diode-react/src/main/scala/diode/react/ReactConnector.scala

```diff
@@ -116,18 +116,19 @@ trait ReactConnector[M <: AnyRef] { circuit: Circuit[M] =>
       }
 
       private def changeHandler(cursor: ModelRO[S]): Unit = {
-        val isMounted = t.isMounted
+        val isMounted = t.isMounted.map(_.getOrElse(false))
```

Comment:
I guess that you are right, I was also considering dropping that check all together since the type of `BackendScope` in 1.0.0 is an alias for `Generic.MountedRoot`... although most likely this does not mean that it is actually mounted...